builtin/mainmenu/dlg_rebind_keys.lua

```diff
@@ -0,0 +1,101 @@
+-- Luanti
+-- SPDX-License-Identifier: LGPL-2.1-or-later
+-- Modified based on dlg_reinstall_mtg.lua
+-- Note that this is only needed for migrating from <5.11 to 5.12.
+
+local doc_url = "https://docs.luanti.org/for-players/controls/"
+
+local function get_formspec(dialogdata)
+	local markup = table.concat({
+		"<big>" .. hgettext("Keybindings changed") .. "</big>",
+		hgettext("The internal input handling system was reworked in Luanti 5.12.0."),
+		hgettext("As a result, your keybindings may have been changed."),
+		hgettext("Refer to the following website for more information:"),
```

Comment:
```suggestion
		"<big>" .. hgettext("Keybindings changed") .. "</big>",
		hgettext("The input handling system was reworked in Luanti 5.12.0."),
		hgettext("As a result, your keybindings may have been changed."),
		hgettext("Check out the key settings or refer to the documentation:"),
```
(very nice would be if we could make "key settings" a link that opens the right tab in settings)